ansible_collections/arista/avd/roles/eos_designs/doc/network-services.md

```diff
@@ -120,6 +120,15 @@ svi_profiles:
     # This setting can be overridden per VRF.
     enable_mlag_ibgp_peering_vrfs: < true | false >
 
+    # EVPN L2 Multicast per SVI | Optional
+    # Configures multicast vxlan vlan multicast group ipv4 address using the following formula: < evpn_l2_multicast.vxlan_multicast_group_ipv4_pool > + < vlan_id - 1 >.
+    # Recommendation to assign a /20 block within 232.0.0.0/16 Source Specific Multicast range.
+    # Enables `redistribute igmp` on the router bgp MAC VRF.
+    evpn_l2_multicast:
+      enabled: < true | false >
+      vxlan_multicast_group_ipv4_pool: < IPv4_address/Mask >
+      vxlan_multicast_group_ipv4_pool_offset: < int >
+
```

Comment:
For L2 EVPN Multicast to work we need also:

Multicast routing must be enabled, and software-forwarding set to SFE
```
router multicast
   ipv4
      routing
      software-forwarding sfe
```

Reply:
I'm not sure about enabling this with this knob, any though on this @emilarista @ClausHolbechArista.
Perhaps we can mention in the description that this is also a requirement along with `underlay_multicast`?

Reply:
So this is a device-wide setting which we should not enable based on a tenant config. So we will need a `evpn_underlay_multicast.enabled: true/false` or similar either as a fabric var or under l3leaf.

Reply:
This is being addressed in PR #1922